component/api.jsonnet

```diff
@@ -120,7 +129,7 @@ local objects = [
             if c.name == 'lieutenant-api' then
               c {
                 image: image,
-                env: [
+                env: mergeEnvVars([
```

Comment:
With `mergeEnvVars` we don't need to patch individual env vars here anymore, you can just add the variables which are patched to your list of additional vars.

Something like

```jsonnet
env: mergeEnvVars(
  super.env,
  com.envList({
    STEWARD_IMAGE: steward_image,
    LIEUTENANT_INSTANCE: lieutenant_instance,
    DEFAULT_API_SECRET_REF_NAME: params.api.default_githost
  } + params.api.env)
)
```

Reply:
Good point. I somehow got confused and thought that wouldn't work :see_no_evil: